### PR TITLE
Publishes the new fec.gov blog post

### PR DESCRIPTION
**Pull request by elainekamlley**
Post for: https://github.com/18F/blog-drafts/issues/633

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/fec-flip-post)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/18f.gsa.gov/fec-flip-post/2017/05/30/the-new-fec/)

Changes proposed in this pull request:
- Publishes The new fec.gov post
- Added images for the post
-

Checklist:
- [x] All images being added have been optimized **_--> [learn more](18f.gsa.gov/styleguide/images/#using-and-optimizing-jpg-and-png) about how to optimize images <--_**

/cc @noahmanger @awfrancisco

### CODE REVIEW

File: _posts/2017-05-30-the-new-fec.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: "The new FEC.gov"
+authors:
+- manger
+tags:
+- fec.gov
+- agency work
+excerpt: "Today, the Federal Election Commission (FEC) unveiled their new website at FEC.gov. This new site is the result of a years-long collaboration with GSA’s 18F and features completely revamped tools for exploring campaign finance data."
```
Comment (noahmanger) [inline _posts/2017-05-30-the-new-fec.md]: Change "Today" to "Last week".

File: _posts/2017-05-30-the-new-fec.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+image: /assets/blog/fec/0-herofec.png
+image_figcaption: "The previous and new FEC website designs"
+---
+
+Today, the Federal Election Commission (FEC) unveiled their new website at FEC.gov. This new site is the result of a years-long collaboration with GSA’s 18F and features completely revamped tools for exploring campaign finance data. It provides user-centered content for understanding the reporting and compliance requirements for people participating in federal elections, redesigned tools for exploring legal resources, and more.
```
Comment (noahmanger) [inline _posts/2017-05-30-the-new-fec.md]: Change "Today" to "Last week"

Comment (noahmanger) [inline _posts/2017-05-30-the-new-fec.md]: Make "FEC.gov" a link.

File: _posts/2017-05-30-the-new-fec.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+</figure>
+
+There are also several major new features we’ll still be working on, including a more powerful search for individual contributions across time, enhancements to data browsing, additional legal resources, and more guidance content. 
+
+Meanwhile, we know that change is hard, and a massive change like this carries a lot of risk. That’s why the entire old experience will continue to be available at [classic.fec.gov](http://classic.fec.gov). That way, users who are used to the old way of navigating will be able to have a fallback in case they need it.
+
+There’s still a lot to do and we will continue working with our partners to ensure that this new FEC.gov meets user needs. We're eager to get feedback from real users and are listening by [email](mailto:webmanager@fec.gov) and on [Github](https://github.com/18f/fec). But we crossed a major milestone this week, and so to our partners, our users, and everyone who helped along the way, all we can say is thank you.
```
Comment (noahmanger) [inline _posts/2017-05-30-the-new-fec.md]: Change "this week" to "last week".

**Comment by noahmanger** [comment]
LGTM.

**Comment by noahmanger** [comment]
Actually, I think we may also wanna add @jenniferthibault to the by-line.

**Comment by noahmanger** [comment]
Update: that's a definitely wanna add Jen as a byline.

File: _posts/2017-05-30-the-new-fec.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
+---
+title: "The new FEC.gov"
+authors:
+- manger
```
Comment (noahmanger) [inline _posts/2017-05-30-the-new-fec.md]: Add @jenniferthibault

Comment (elainekamlley) [inline _posts/2017-05-30-the-new-fec.md]: 👍

**Pull request merged**